app/_src/gateway/install/linux/ubuntu.md

```diff
@@ -103,7 +103,7 @@ Install {{site.base_gateway}} on Ubuntu from the command line.
 
 1. Download the Kong package:
 
-{% assign ubuntu_flavor = "jammy" %}
+{% assign ubuntu_flavor = "$(lsb_release -sc)" %}
```

Comment:
I'm not sure this is a good idea. It means non-jammy / focal installs will fail as there are no packages built https://cloudsmith.io/~kong/repos/gateway-37/packages/?q=distribution:ubuntu